src/main/assembly/dist/examples/abr-type/example1.xml

```diff
@@ -21,9 +21,7 @@
          xmlns:dc="http://purl.org/dc/elements/1.1/"
          xmlns:dct="http://purl.org/dc/terms/"
          xmlns:abr="http://www.den.nl/standaard/166/Archeologisch-Basisregister/"
-         xsi:schemaLocation=
-                 "http://easy.dans.knaw.nl/schemas/md/ddm/ https://easy.dans.knaw.nl/schemas/md/ddm/ddm.xsd
-    http://www.den.nl/standaard/166/Archeologisch-Basisregister/ https://easy.dans.knaw.nl/schemas/vocab/abr-type.xsd">
```

Comment:
is this no longer necessary?

same for the other instances in this whole diff where you removed `schemaLocation` entries

Reply:
👍